obspy/core/inventory/response.py

```diff
@@ -1663,6 +1663,42 @@ def get_sacpz(self):
         paz = self.get_paz()
         return paz_to_sacpz_string(paz, self.instrument_sensitivity)
 
+    @staticmethod
+    def from_paz(zeros, poles, stage_gain,
+                 stage_gain_frequency=1.0, input_units='VOLTS',
+                 output_units='M/S', normalization_frequency=None,
+                 pz_transfer_function_type='LAPLACE (RADIANS/SECOND)',
+                 normalization_factor=1.0):
+        """
+        Takes in lists of complex poles and zeros and returns a Response with
+        those values defining its only stage.
```

Comment:
If possible, please rephrase the docstring into
```python
"""
Single line description
   <- Empty Line
Longer description.

...
"""
```

Then it will render nicer in the sphinx documentation.